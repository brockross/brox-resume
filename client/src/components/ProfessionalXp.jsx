import React from 'react';

const ProfessionalXp = (props) => {
  return (
    <div>
      <h3>Professional Experience</h3>
      <ul>
        <li>Quapp.co</li>
        <li>Brave Living</li>
      </ul>
    </div>
  )
}

export default ProfessionalXp;