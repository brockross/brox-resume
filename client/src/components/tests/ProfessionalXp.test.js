import React from 'react';
import { shallow } from 'enzyme';

import ProfessionalXp from  '../ProfessionalXp.jsx';

describe('ProfessionalXp', () => {
  it('should render correctly', () => {
    const component = shallow(<ProfessionalXp/>);

    expect(component).toMatchSnapshot();
  });
});