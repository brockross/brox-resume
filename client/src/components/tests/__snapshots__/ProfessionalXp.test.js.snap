// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfessionalXp should render correctly 1`] = `
<div>
  <h3>
    Professional Experience
  </h3>
  <ul>
    <li>
      Quapp.co
    </li>
    <li>
      Brave Living
    </li>
  </ul>
</div>
`;
